R/rocco.R

```diff
@@ -26,25 +29,31 @@
 #'   # The below will simply create a static HTML site without opening it.
 #'   rocco("/path/to/package", output_dir = "/my/html/dir", browse = FALSE)
 #' }
-rocco <- function(directory, output_dir = tempdir(), browse = interactive(), gh_pages = FALSE) {
+rocco <- function(directory, output_dir = tempdir(), browse = interactive(),
+  rocco = TRUE, staticdocs = TRUE, gh_pages = FALSE) {
   if (missing(directory)) directory <- "."
```

Comment:
I would check that the package is installed if `staticdocs = TRUE` since it is in Suggests.